docs/limitations.asciidoc

```diff
@@ -0,0 +1,12 @@
+[[limitations]]
+= Limitations
+
+[partintro]
+--
+This section documents certain limitations of Kibana. These limitations exist by design today but may be removed in the future, as Kibana evolves.
```

Comment:
I'm not a big fan of meta info like "This section documents...". It's pretty redundant to say that a section titled Limitations documents limitations.  I'd probably drop the first sentence and modify the second to just say: "Kibana currently has the following limitations."